### DISCUSSION TITLE
Copy link for missing c++ runtime to clipboard.

### DISCUSSION THREAD

**Discussion by Kapcode** [transmission/transmission#6360] [Ideas]
![Screenshot 2023-12-09 142221](https://github.com/transmission/transmission/assets/6187749/9a6e774c-e997-400f-99b7-dd10964dbdee)

as of right now if you are missing the required library, you have to open a browser, and type it manually, or search for it in a search engine. If it was changed to a hyperlink, or a button to set clipboard contents, would be easy to implement, and save time :)

**Comment by mikedld** [comment]
Currently impossible, see #4802.
